src/mono/netcore/System.Private.CoreLib/src/Mono/RuntimeStructs.cs

```diff
@@ -1,12 +1,13 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
```

Comment:
In the past, we'd been leaving out this middle line when adding the license header to files from Mono.
@richlander, can you comment on what we should be doing moving forward here?

Reply:
Either the two line or three line version is fine. And yes, its the middle line that can be skipped.

Reply:
We'll leave the 3 lines in there. Resolving comment.

Reply:
Correction ... This is the 2-line version we decided on:

```csharp
// Licensed to the .NET Foundation under one or more agreements.
// The .NET Foundation licenses this file to you under the MIT license
```

Reply:
@fanyang-mono - please update to the 2-line version. Thank you.

Reply:
Rich said that either two line or three line version is fine. Why do we have to use the two line version?

Reply:
@richlander Just to clarify, the 3-line version is fine as well, right?

Reply:
I'm currently in the process of trying to move the entire repo to the 2-line version.

Reply:
I see. I will make my change into 2-line version then.

Reply:
@stephentoub I updated the copyright to 2-line version. Could you please review?